fmriprep/workflows/bold/base.py

```diff
@@ -664,18 +663,22 @@ def init_func_preproc_wf(bold_file):
         if not multiecho:
             workflow.connect([
                 (bold_split, bold_std_trans_wf, [
-                    ('out_files', 'inputnode.bold_split')])
+                    ('out_files', 'inputnode.bold_split')]),
+                (bold_sdc_wf, bold_std_trans_wf, [
+                    ('outputnode.out_warp', 'inputnode.fieldwarp')]),
+                (bold_hmc_wf, bold_std_trans_wf, [
+                    ('outputnode.xforms', 'inputnode.hmc_xforms')]),
             ])
         else:
-            split_opt_comb = bold_split.clone(name='split_opt_comb')
             workflow.connect([
-                (bold_t2s_wf, split_opt_comb, [
-                    ('outputnode.bold', 'in_file')]),
                 (split_opt_comb, bold_std_trans_wf, [
-                    ('out_files', 'inputnode.bold_split')
-                ])
+                    ('out_files', 'inputnode.bold_split')])
             ])
 
+            # Already applied in bold_bold_trans_wf, which inputs to bold_t2s_wf
+            bold_std_trans_wf.inputs.inputnode.fieldwarp = 'identity'
+            bold_std_trans_wf.inputs.inputnode.hmc_xforms = 'identity'
```

Comment:
@tsalo Similarly for bold_std_trans_wf

Reply:
Because that's the specific issue that this is fixing (i.e., #2236), right? Therefore, distortion correction on multi-echo data should have been exaggerated before the fix and should be better now.

Reply:
Yes.